internal/vrf/create.go

```diff
@@ -0,0 +1,80 @@
+package vrf
+
+import (
+	"fmt"
+	"strconv"
+	"github.com/packethost/packngo"
+	"github.com/spf13/cobra"
+)
+
+
+func (c *Client) Create() *cobra.Command {
+	var (
+	    metro string
+	    name string
+	    description string
+	    ipRanges []string
+	    localASN int
+        )
+
+	// createVRFCmd represents the creatVRF command
+	createVRFCmd := &cobra.Command{
+		Use:   `create vrf <my_vrf> [-m <metro_code>] [-AS int] [-I str] [-d <description>]`,
+		Short: "Creates a virtual network.",
+		Long:  "Creates a VRF",
+		Example: `# Creates a VRF, metal vrf create `,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			req := &packngo.VRFCreateRequest{
+				Metro:     metro,
```

Comment:
Metro in which to create the VLAN. Mutually exclusive with Facility. We are supporting Facility too.

Reply:
VRFs can only be created in metros. The underlying API for this feature doesn't support facility.